bitcoin/secp256k1_main.py

```diff
@@ -239,23 +236,17 @@ def multiply(s, pub, usehex, rawpub=True):
     of the scalar s.
     ('raw' options passed in)
     '''
-    newpub = secp256k1.PublicKey(pub, raw=rawpub, ctx=ctx)
-    #see note to "tweak_mul" function in podle.py
-    res = secp256k1._tweak_public(newpub,
-                                   secp256k1.lib.secp256k1_ec_pubkey_tweak_mul,
-                                   s)
-    return res.serialize()
+    res = coincurve.PublicKey(pub)
+    res.multiply(s, update=True)
```

Comment:
I believe update only changes the PublicKey object 'res', and doesn't effect 'pub'. Changed it anyway to remove the confusion.